.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
```

Comment:
Just a note that Stackbit projects tend to use npm over yarn. Just noting. I wouldn't make any changes with this right now and instead stay focused on the content.